tests/system/src/test/java/test/robot/javafx/embed/swing/NonFocusableJFXPanelTest.java

```diff
@@ -37,10 +37,35 @@
 import javafx.embed.swing.JFXPanel;
 import javafx.scene.Scene;
 import javafx.scene.control.Button;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.Arguments;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
```

Comment:
I promised myself I wasn't going to look at the imports, but... Does Eclipse  really mix static and non-static imports? And are any of the static imports actually used? I suspect not in both cases, so you might just need to have Eclipse fix up your imports.

Reply:
comment refers to an earlier version which has been fixed since then.

to answer Eclipse question, the imports are sorted like this: static come first, then I think it's in alphabetical order.